### DISCUSSION TITLE
Users Permission

### DISCUSSION THREAD

**Discussion by pelucapampa** [chatwoot/chatwoot#8413] [Feature Requests]
I think chatwoot have a Permission Group to ONLY see Agent asigned conversation.

It's possible to setup with check on User profile and then, only can see your assigned conversation.

**Comment by rgfpy** [comment]
i think like that, and need custom roles to assign users

**Comment by hereck** [comment]
The chatawoot blog says it has a team configuration to manage permission, but it still doesn't work.

https://www.chatwoot.com/blog/team-permissions
